docs/index.md

```diff
@@ -22,14 +22,25 @@ For details about the service API see the [OpenAPI specification].
 ARGUMENTS
 ---------
 
-          --help                Show help message
-          --version             Show version of this program
-
-    Subcommand: run-service - Starts the EASY Bag Store as a daemon that services
-                              HTTP requests
-          --help   Show help message
+      -h, --help      Show help message
+      -v, --version   Show version of this program
+    
+    Subcommand: run-service - Starts EASY Deposit Api as a daemon that services HTTP requests
+      -h, --help   Show help message
+    ---
+    
+    Subcommand: change-state - Changes the state of a deposit, when changing to SUBMITTED just the state is changed, the rest of the submit-cycle is not started
+      -d, --description  <arg>   A desription of the new state
+          --doUpdate             without this argument only the current status is
+                                 shown in json format
```

Comment:
> without this argument only the current status is shown in json format

Is that really true? Hopefully it will also show the state you would have changed to if this argument was provided.